Pipfile

```diff
@@ -4,90 +4,89 @@ verify_ssl = true
-opentelemetry-instrumentation-fastapi = "==0.45b0"
-opentelemetry-instrumentation-logging = "==0.45b0"
-opentelemetry-instrumentation-sqlite3 = "==0.45b0"
-opentelemetry-instrumentation-tortoiseorm = "==0.45b0"
-opentelemetry-instrumentation-urllib = "==0.45b0"
-opentelemetry-instrumentation-wsgi = "==0.45b0"
+aio-pika = "==9.4.3"
+azure-storage-blob = "==12.22.0"
+taskiq-fastapi = "==0.3.2"
+taskiq-redis = "==1.0.0"
+taskiq-aio-pika = "==0.4.1"
+sqlalchemy-utils = "==0.41.2"
+typer = "==0.12.5"
+aiofiles = "==24.1.0"
+opentelemetry-api = "==1.27.0"
+opentelemetry-sdk = "==1.27.0"
+opentelemetry-exporter-otlp = "==1.27.0"
+opentelemetry-distro = "==0.48b0"
+opentelemetry-instrumentation = "==0.48b0"
+opentelemetry-instrumentation-asgi = "==0.48b0"
+opentelemetry-instrumentation-asyncio = "==0.48b0"
+opentelemetry-instrumentation-dbapi = "==0.48b0"
+opentelemetry-instrumentation-fastapi = "==0.48b0"
+opentelemetry-instrumentation-logging = "==0.48b0"
+opentelemetry-instrumentation-sqlite3 = "==0.48b0"
+opentelemetry-instrumentation-tortoiseorm = "==0.48b0"
+opentelemetry-instrumentation-urllib = "==0.48b0"
+opentelemetry-instrumentation-wsgi = "==0.48b0"
+opentelemetry-semantic-conventions = "==0.48b0"
+opentelemetry-test-utils = "==0.48b0"
+opentelemetry-util-http = "==0.48b0"
 opentelemetry-propagator-aws-xray = "==1.0.1"
-opentelemetry-sdk = "==1.24.0"
-opentelemetry-semantic-conventions = "==0.45b0"
-opentelemetry-test-utils = "==0.45b0"
-opentelemetry-util-http = "==0.45b"
-opentelemetry-exporter-otlp = "==1.24.0"
-opentelemetry-sdk-extension-aws = "==2.0.1"
-nh3 = "==0.2.17"
+opentelemetry-sdk-extension-aws = "==2.0.2"
+nh3 = "==0.2.18"
 pymongo = "*"
 
 [dev-packages]
 # Nobody knows for what its needed
-ipdb = "~=0.13"
-pudb = "~=2022.1"
+ipdb = "==0.13.13"
+pudb = "==2024.1.2"
 # Linters and Formatters
-pre-commit = "~=2.7.1"
-ruff = "=0.6.2"
+pre-commit = "==3.8.0"
+ruff = "==0.6.4"
 # Tests
-allure-pytest = "*"
-pydantic-factories = "~=1.17.0"
-pytest = "~=7.1"
+allure-pytest = "==2.13.5"
+pydantic-factories = "==1.17.3"
+pytest = "==8.3.3"
 pytest-asyncio = "~=0.19"
-pytest-cov = "~=3.0"
-pytest-env = "~=0.8.1"
-pytest-lazy-fixture = "~=0.6"
-pytest-mock = "~=3.8"
+pytest-cov = "==5.0.0"
+pytest-env = "==1.1.4"
+pytest-lazy-fixtures = "==1.1.1"
+pytest-mock = "==3.14.0"
 nest-asyncio = "==1.6.0"
-gevent = "~=23.9"
+gevent = "==24.2.1"
 # MyPy
-mypy = "==1.10.0"
+mypy = "==1.11.2"
 types-passlib = "==1.7.7.20240819"
-types-python-dateutil = "==2.9.0.20240821"
+types-python-dateutil = "==2.9.0.20240906"
 types-python-jose = "==3.3.4.20240106"
 typing-extensions = "==4.12.2"
 types-requests = "==2.32.0.20240712"
 types-pytz = "==2024.1.0.20240417"
 types-aiofiles = "==24.1.0.20240626"
 types-cachetools = "==5.5.0.20240820"
 # https://github.com/sqlalchemy/sqlalchemy/issues/7714
-greenlet = "~=2.0.1"
+greenlet = "==3.0.3"
 # JSONLD deps only for dev
 reproschema = "*"
 cachetools = "==5.3.0"
```

Comment:
yes, there are a lot of packages updates, but just a change from aioredis to redis-py is a critical one, the others changes will not bring much impact

some changes in code are because of theses updates, but there are other changes to fix some warnings, specially related to queries

But we still have some big changes pending, like pydantic, sqlalchemy and fastapi

pytests tests 79% of API, E2E tests more, I believe it is enough, but only after smoke tests will we be sure of it